indica/bayesmodels.py

```diff
@@ -73,17 +83,25 @@ def _build_bckc(self, params: dict, **kwargs):
     def _ln_likelihood(self):
         ln_likelihood = 0
         for key in self.quant_to_optimise:
-            # TODO: What to use as error?  Assume percentage error if none given...
-            # Float128 is used since rounding of small numbers causes
-            # problems when initial results are bad fits
-            model_data = self.bckc[key].values.astype("float128")
+            # Float128 since rounding of small numbers causes problems
```

Comment:
Question:
would rounding the model data be a better/cleaner solution instead of changing type?